### ISSUE TITLE
Built 0.19.1 from source, get the warning "This is a pre-release..." developers forgot to remove it.

### ISSUE THREAD

**Issue by mrobinton** [bitcoin/bitcoin#18355]
**Expected behavior**

**Actual behavior**

**To reproduce**

**System information**

**Comment by mrobinton** [comment]
Subject says it all

**Comment by MarcoFalke** [comment]
See commit 58ba7c314d552cea8cb024960a8504577aee586f, which says ` define(_CLIENT_VERSION_IS_RELEASE, true)` on the tangent line.

What commit are you compiling exactly?

**Comment by mrobinton** [comment]
I downloaded the master tree from github. The link from bitcoin.org indicated it was the new release of 0.19.1. Someone should tell the appropriate person at bitcoin.org to link to the commit instead of the master tree.
When I grep CLIENT_VERSION_IS_RELEASE, it is set to false.
Latest commit d402c1e 5 hours ago
I see now that a few patches have been added since 58ba7c3
I'm not using the wallet so I'll go ahead and use the daemon I compiled. I'm running 3 of them to support the network at each of the locations where we have network service.
